Firestore/Example/Podfile

```diff
@@ -3,22 +3,21 @@
 # corresponding podspec's.
 pod 'Firebase/Core', '4.11.0'
 
-use_frameworks!
+#use_frameworks!
 platform :ios, '8.0'
 
 target 'Firestore_Example' do
   pod 'FirebaseAuth', :path => '../../'
   pod 'FirebaseCore', :path => '../../'
   pod 'FirebaseFirestore', :path => '../../'
+  pod 'GoogleTest', :podspec => 'Tests/GoogleTest/GoogleTest.podspec'
```

Comment:
I suspect something like this change might be needed, but this pod dependency change made no difference.

Reply:
This is wrong though: the example app has no dependency on `'GoogleTest'`, only the tests do.